content/projects/gsoc/2021/projects/conventional-commits-plugin.adoc

```diff
@@ -45,26 +44,29 @@ For a existing project released at `1.0.0`, the following commit messages would
 
 ==== Project Details
 
-This project idea proposes to implement a Jenkins plugin which will allow the `nextVersion` to be determined from the commit log.
+The conventional commits plugin is a Jenkins plugin to automatically determine the next semantic version for a git repository using the Conventional Commits (C.C) system.
 
-The project requires the student to start with plugin development from scratch and then work with understanding how to integrate this plugin into a Jenkins pipeline.
+Taking in various sources of input, like the commit history of the repository, the latest tagged version, and the version information read from various configuration files of the project, the plugin outputs the next semantic version.
 
-In the beginning, the student can work on understanding the Jenkins plugin development by writing code for creating a simple Build Step which determines the current version from the latest tag.
+For more details please see the following Resources section.
```

Comment:
I would be interested to now if it work with the scripted pipeline. The Github README only shows examples of the declarative pipeline. My guess is yes but without an actual example, I'll have to download it and try, but I would prefer to know just by reading the documentation. Thank you.

Reply:
Yes! I will add that, even I would need to try with a scripted pipeline. 
But, would it make sense to add that example in the project Readme instead?

Reply:
+1 for README

Reply:
Yes the README is fine. Make sure you provide and example for both if supported by both, otherwise the unaware reader will think it only works in Declarative and not in Scripted.

Reply:
In here you can say something like "Examples for the Declarative and the Scripted Pipeline can be found in the README".

Reply:
Done :)

Thanks a lot @martinda